### PR TITLE
Paper examples

### PR DESCRIPTION
**Pull request by JGrantHill**
Adds examples from the paper.

### CODE REVIEW

File: examples/multi-molecule/multi-molecule.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+bo.set_backend('psi4')
+bo.set_tmp_dir('/home/me/scr/')
+
+mb = MolecularBasis(name="double")
+list_of_mols = ['water', 'methane', 'methanol', 'formaldehyde', 'oxygen']
+mol_objs = []
+for molecule in range(len(list_of_mols)):
+	mol_objs.append(bo.molecule.Molecule.from_xyz(list_of_mols[molecule]+'.xyz', 
+           name=list_of_mols[molecule]))
+	mb.add_molecule(mol_objs[molecule])
```
Comment (robashaw) [inline examples/multi-molecule/multi-molecule.py:15]: I would tidy this up as 
```python
list_of_mols = ['water', 'methane', 'methanol', 'formaldehyde', 'oxygen']
mol_objs = [
      bo.molecule.Molecule.from_xyz(mol+'.xyz', name=mol)
      for mol in list_of_mols
]
mb = MolecularBasis(name="double", molecules=mol_objs)
```

**Review by robashaw — APPROVED** [review]
Generally approve merging, just one suggestion for you to look at

File: examples/multi-molecule/multi-molecule.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+from basisopt.opt.strategies import Strategy
+from basisopt.util import bo_logger
+
+bo.set_backend('psi4')
+bo.set_tmp_dir('/home/me/scr/')
+
+mb = MolecularBasis(name="double")
+list_of_mols = ['water', 'methane', 'methanol', 'formaldehyde', 'oxygen']
+mol_objs = []
+for molecule in range(len(list_of_mols)):
```
[the remark this answers is not in the document]
Comment (robashaw) [reply examples/multi-molecule/multi-molecule.py:12]: Yes, the index to a list of names. The molecules are then created by referencing the list of names. That's why I think it's clearer to make and pass the molecule objects directly.

**Comment by JGrantHill** [comment]
I've included the tidying-up suggestion in my latest commit, hopefully ready to merge.
Need to remember to update the manuscript draft to reflect the changes.

**Pull request merged**